ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -298,7 +297,7 @@ protected:
   }
 
 private:
-  using EASVariant = EAS::Variants<Geometry>::type;
+  using EASVariant = typename EAS::Variants<Geometry>::type;
```

Comment:
In c++ 20 the paper https://wg21.link/P0634R3 got accepted which makes typename obsolete in some contexts

Reply:
See the exaample with using therein

Reply:
I can not create a link to the example but therin search for `using Ptr = PtrTraits<T>::Ptr;`